Header header
float64[]                  encoders
float64[]                  forces
float64[]                  fk
float64[]                  motor_cmd
uint64[]		   time_stamps
